llmv_models.py

```diff
@@ -16,6 +17,22 @@
 
 log = getLogger(__name__)
 
+class LLMVChatPair(db.Model):
+    __tablename__ = 'llmv_chat_pair'
+    __table_args__ = {'extend_existing': True}
```

Comment:
Is this table defined elsewhere?

https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Table.params.extend_existing